kuma/users/jinja2/users/email/payment_received/plain.ltxt

```diff
@@ -0,0 +1,10 @@
+{% autoescape false -%}
+{{ _('Hello from MDN!') }}
+
+{{ invoice_pdf }}
+
+Cancel any time. To cancel your subscription, please go to the manage monthly subscriptions page: {{ manage_subscription_url }}.
+If you have questions, please read the FAQ ({{ faq_url }}) or you can also contact support: {{ contact_email }}
+
+{{ _('Thank you for using MDN!') }}
+{% endautoescape -%}
```

Comment:
I'm not going to review this. I'm sure you got this from @atopal or something. 
By the way, all the links work as expected. I clicked on everything in the email printed on `stdout`.